models/jachin_bmk.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import (Integer, Column, String, Date)
+
+from models.config import Base, db_engine
+
+
+class BmkTable(Base):
+    __tablename__ = 'bmk_table'
+    __table_args__ = {'extend_existing': True}
+    bmk_id = Column('bmk_id', Integer, primary_key=True)
```

Comment:
Leave a single white space before starting to define the table attributes.
This is just for code readability

And leave a single whitespa at every end of every module.
Awesome!